### DISCUSSION TITLE
voice audio to byte[]

### DISCUSSION THREAD

**Discussion by JackRumboll** [StereoKit/StereoKit#781] [General]
I'm looking for a way to send my microphone data over the network but for that to work, I need the data to be in bytes so I can send the bytes. I also need a way to turn the bytes back into audio so that the other people's microphone data can play in my scene.

**Comment by maluoi** [comment]
Check out the example code for [`Sound.ReadSamples`](https://stereokit.net/Pages/StereoKit/Sound/ReadSamples.html), this'll give you access to a float[] which should cover what you need! Similarly, to turn that data back into sound later on, see [`Sound.WriteSamples`](https://stereokit.net/Pages/StereoKit/Sound/WriteSamples.html).

There's a few more bits of sample code related to creating audio [over in this demo](https://github.com/StereoKit/StereoKit/blob/master/Examples/StereoKitTest/Demos/DemoSound.cs) as well!
